WcaOnRails/app/views/static_pages/about.html.erb

```diff
@@ -1,50 +1,45 @@
-<% provide(:title, 'About the WCA') %>
+<% provide(:title, t('about.title')) %>
 
 <div class="container">
   <h1><%= yield(:title) %></h1>
 
-  <h3>Mission</h3>
+  <h3><%= t('about.mission.title') %></h3>
 
-  <p>The World Cube Association governs competitions for all puzzles labelled as Rubik puzzles, and all other puzzles that are played by twisting the sides, so-called 'twisty puzzles'. Most famous of these puzzles is the Rubik's Cube, invented by professor Rubik from Hungary. A selection of these puzzles are chosen as official events of the WCA.</p>
+  <% t('about.mission').values.each do |paragraph| %>
+    <p>
+      <%= raw paragraph unless paragraph == 'Mission' %>
```

Comment:
Assuming that `t('about.mission')` is a hash then you can simply do `t('about.mission').each do |key, paragraph|`